schema.sql

```diff
@@ -112,4 +112,23 @@ CREATE TABLE VISITS(
     VETS_ID INT,
     CONSTRAINT VETS_ID FOREIGN KEY(VETS_ID) REFERENCES VETS(ID),
     DATE_OF_VISIT DATE
-);
\ No newline at end of file
+);
+
+
+
+-------------------------------- WEEK 02: DAY 01 --------------------------------
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- SELECT COUNT(*) FROM visits where animal_id = 4;
+CREATE INDEX ANIMALS_ID_INDEX ON VISITS(ANIMALS_ID);
+
+-- SELECT * FROM visits where vet_id = 2;
+CREATE INDEX VETS_ID_INDEX ON VISITS(VETS_ID);
+
+-- SELECT * FROM owners where email = 'owner_18327@mail.com';
+CREATE INDEX EMAIL_INDEX ON OWNERS(EMAIL);
```

Comment:
- [ ] You were supposed to include a pair of screenshots in the PR description with the results of the  EXPLAIN ANALYZE command for each of the above queries (in total it should be 6 screenshots):

:point_right: The first screenshot should show how much time the query takes BEFORE the improvements you introduced.
:point_right: The second screenshot should show how much time the query takes AFTER the improvements you introduced.

Reply:
Really sorry for that, Completely forgot to attach the screenshots. Let me do it quickly.